src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/UpdateTableOperation.java

```diff
@@ -0,0 +1,91 @@
+import com.datastax.oss.driver.api.querybuilder.update.UpdateStart;
+import com.datastax.oss.driver.api.querybuilder.update.UpdateWithAssignments;
+import io.smallrye.mutiny.Uni;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandResult;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandStatus;
+import io.stargate.sgv2.jsonapi.api.request.DataApiRequestInfo;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.QueryExecutor;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.TableSchemaObject;
+import io.stargate.sgv2.jsonapi.service.operation.query.UpdateValuesCQLClause;
+import io.stargate.sgv2.jsonapi.service.operation.query.WhereCQLClause;
+import java.util.*;
+import java.util.function.Supplier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * POC Updates a Table using a CQL UPDATE statement.
+ *
+ * <p>TODO: marking POC because this is the first implementation.
+ */
+public class UpdateTableOperation extends TableMutationOperation {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(UpdateTableOperation.class);
+
+  private final UpdateValuesCQLClause updateValuesCQLClause;
+  private final WhereCQLClause<Update> whereCQLClause;
+
+  public UpdateTableOperation(
+      CommandContext<TableSchemaObject> commandContext,
+      UpdateValuesCQLClause updateValuesCQLClause,
+      WhereCQLClause<Update> whereCQLClause) {
+    super(commandContext);
+
+    this.updateValuesCQLClause =
+        Objects.requireNonNull(updateValuesCQLClause, "updateValuesCQLClause must not be null");
+    this.whereCQLClause = Objects.requireNonNull(whereCQLClause, "whereCQLClause must not be null");
+  }
+
+  @Override
+  public Uni<Supplier<CommandResult>> execute(
+      DataApiRequestInfo dataApiRequestInfo, QueryExecutor queryExecutor) {
+
+    // Start the select
+    UpdateStart updateStart =
+        update(
+            commandContext.schemaObject().tableMetadata.getKeyspace(),
+            commandContext.schemaObject().tableMetadata.getName());
+
+    // Update the columns
+    List<Object> positionalValues = new ArrayList<>();
+    UpdateWithAssignments updateWithAssignments =
+        updateValuesCQLClause.apply(updateStart, positionalValues);
+
+    // HACK AARON - do not know how to get on the correct interface here
+    Update update = (Update) updateWithAssignments;
+    // Add the where clause
+    update = whereCQLClause.apply(update, positionalValues);
+
+    var statement = update.build(positionalValues.toArray());
+    LOGGER.warn("UPDATE CQL: {}", update.asCql());
+    LOGGER.warn("UPDATE VALUES: {}", positionalValues);
+
+    return queryExecutor
+        .executeWrite(dataApiRequestInfo, statement)
+        .onItem()
+        .transformToUni(UpdateTableOperation::getResultSupplier);
+  }
+
+  private static Uni<Supplier<CommandResult>> getResultSupplier(AsyncResultSet result) {
```

Comment:
The modified and matched count set to `1`.  The code flow looks fine, but these needs to be revisited. As clarified in another class is the POC scope limited to 1 row operations?